src/main/java/com/example/demo/services/DefaultItemService.java

```diff
@@ -77,12 +79,18 @@ public ItemModel getItem(int id) throws AuctionAppException{
 
 	@Override
 	public ItemModel addItem(ItemModel itemModel,User user) throws AuctionAppException {
+		if(user.getMerchantId()!=null&&user.getMerchantId().isBlank()) {
+			Map<String,String> problems=new HashMap<>();
+			problems.put("User", "Userm must have seller account, PayPal integration required");
```

Comment:
I had to merge PayPal into the Delete branch because deleting requires checking if item is paid...as you know I stuck as PayPal branch and couldn't afford to lose time while fixing PayPal